p100/tasks/main.yml

```diff
@@ -0,0 +1,37 @@
+- name: Create presentation client user
+  user:
+    name: p
+    create_home: true
+    home: /p
+    uid: 17263
+
```

Comment:
would be nice if the shell was /usr/bin/bash  (vs dash).

also in the root_password branch i had added this user along with:
```
    - name: add run script for p
      ansible.builtin.copy:
        src: p.run
        dest: /p/run
        owner: p
        group: p
        mode: '0755'
    - name: add .xsessionrc script for p
      ansible.builtin.copy:
        content: |
          /p/run
          exit 0
        dest: /p/.xsessionrc
        owner: p
        group: p
        mode: '0755'
```

if you include those here I can remove p100 from the root_password branch.

Reply:
along with files/p.run